### ISSUE TITLE
Replacing "item" text in chat output?

### ISSUE THREAD

**Issue by nazzyz** [The-Fireplace-Minecraft-Mods/Grand-Exchange#9]
This is not really an issue, its more of a cosmetic request but... in Grand Economy, I'm going to attempt to use Ordinary Coins as a tangible currency.  This seems to be working so far where I have mobs drop coins as part of their loot.  Players can pick up the coins then use /convert to add the coin value to their personal wallet.
the output in chat says: You have exchanged 1 _ordinarycoins:coinBronze_ for 1 each, earning a total of 1. Your balance is...
Is there a way to make it not say the item name but rather a title of some sort, like _You have exchanged 1 **Bronze Coin**..._ or something?

In the conversion_items.json config, I'm giving the item a value:
{
      "item": "ordinarycoins:coinBronze",
      "meta": 0,
      "value": 1
    },
the output text is just reading from "item"
I just feel like there is another command or something that I can add that says tell "item" to output "title" in chat screen, I just am not sure if that's possible?
thanks for looking into it, your collection of economy mods look fantastic so far, great job and ideas!

**Comment by eleksploded** [comment]
What version would you want this for?

**Comment by The-Fireplace** [comment]
1.12.2

**Comment by nazzyz** [comment]
I was trying a few lines of commands I kind of know but I couldn't get it
to replace text.  I was putting together a 1.12.2 server so I would say for
that version.  Most people still play 1.12.2 so that would probably be most
beneficial.  Thanks for looking into this!

**Comment by nazzyz** [comment]
I was trying a few lines of commands I kind of know but I couldn't get it
to replace text.  I was putting together a 1.12.2 server so I would say for
that version.  Most people still play 1.12.2 so that would probably be most
beneficial.  Thanks for looking into this!

**Comment by The-Fireplace** [comment]
Alright, so a potential complication here is that if the name and not the id is shown, users may have no idea, for example, which Bronze Ingot in a modpack with 5 different versions they are looking at. This becomes a problem when mods are involved that use OreDictionary poorly or not at all, resulting in recipes that can only be crafted with one of the 5 types of the Bronze Ingot.

**Comment by The-Fireplace** [comment]
In addition, if the name is shown and not the ID, then users would expect that they could type exactly that name to accept an offer, which would mean 1.) Ambiguity if the previously mentioned case is reached and 2.) accepting spaces in the name that gets typed, which is a problem because it would then result in having to either force users to use quotation marks, causing confusion for users, or accepting a variable number of arguments, which becomes problematic for numerous reasons.

So in short, unless any ideas are presented that mitigate all of these issues without losing the benefit of showing the name, this one can't reasonably happen.
